addons/web/static/src/views/form/form_controller.js

```diff
@@ -93,6 +95,15 @@ export class FormController extends Component {
         this.router = useService("router");
         this.user = useService("user");
         this.viewService = useService("view");
+        this.ui = useService("ui");
+        const renderOnResize = useDebounced(this.render, 101);
+        useEffect(
+            () => {
+                window.addEventListener("resize", renderOnResize);
+                return () => window.removeEventListener("resize", renderOnResize);
+            },
+            () => []
+        );
```

Comment:
I hate this. We really need to make ui into a reactive object so components can rerender when things change without doing this.

Reply:
(ended up going with a bus, we'll go with a reactive at some point in the future)